integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -141,16 +139,19 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
           targetCpu = masterCpu;
           targetMem = masterMem;
           mMasterHostname = offer.getHostname();
-          mMasterLaunched = true;
+          mIsMasterTask = true;
```

Comment:
since mIsMasterTask is only used to set the task name you could replace this with

``mTaskName = sConf.get(Constants.INTEGRATION_MESOS_TACHYON_MASTER_NAME)``